gen3.biodatacatalyst.nhlbi.nih.gov/etlMapping.yaml

```diff
@@ -148,6 +148,6 @@ mappings:
     injecting_props:
       subject:
         props:
-          - name: subject_id
+          - name: _subject_id
             src: id
```

Comment:
i believe there needs to be a `fn: set` here -- @mpingram can you confirm?

Reply:
Yes, great catch -- I think it's actually ok to leave out the `fn: set` fix in this hotfix, I think the issue you noticed here https://cdis.slack.com/archives/CN0E43KQC/p1602094727306600?thread_ts=1602093431.304700&cid=CN0E43KQC has been in bdcat prod for a long time and doesn't cause any immediate issues.

So I was thinking that we can put the `fn: set` fix in preprod and promote it to prod as part of our normal release process. @ZakirG @themarcelor can I get your opinion on this call though?